server/routes/v1/model.ts

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
We should fix what's causing this first, and if we really can't resolve then use a "next-line" one instead of ignoring the rule for the full file

Reply:
We really shouldn't be ignoring our eslint rules at all, unless absolutely necessary. The whole point of having them is to set a standard that we want our code to reach.